### PR TITLE
New Formplayer: Icons mimic mobile + responsive

### PR DESCRIPTION
**Pull request by biyeun**
@benrudolph @wspride 
cc @orangejenny 

responsive-cloudcare-icons
app-preview-icons

### CODE REVIEW

File: corehq/apps/cloudcare/templates/cloudcare/formplayer_home.html [hunk 1]
```diff
@@ -57,9 +57,7 @@
 {% endblock %}
 
 {% block js %} {{ block.super }}
-    {% compress js %}
```
Comment (benrudolph) [inline corehq/apps/cloudcare/templates/cloudcare/formplayer_home.html]: huh weird

**Comment by benrudolph** [comment]
looks great!! 👍 on ✅

**Comment by benrudolph** [comment]
@wpride (last time i check @wspride was deprecated)

**Comment by wpride** [comment]
wow @biyeun looks really amazing 🎩

**Pull request merged**